src/HDInsight/HDInsight.Test/UnitTests/GetPropertiesTests.cs

```diff
@@ -37,36 +37,37 @@ public GetPropertiesTests(Xunit.Abstractions.ITestOutputHelper output)
             };
         }
 
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
-        public void CanGetProperties()
-        {
-            var features = new string[] { "feature1", "feature2" };
-            var versions = new Dictionary<string, VersionsCapability> { { "key", new VersionsCapability() } };
-            var vm = new Dictionary<string, VmSizesCapability> { { "key1", new VmSizesCapability() } };
-            var regions = new Dictionary<string, RegionsCapability> { { "eastus", new RegionsCapability() } };
-            var propertiesResponse = new CapabilitiesResponse
-            {
-                Features = features,
-                Versions = versions,
-                VmSizes = vm,
-                Regions = regions
-            };
-            hdinsightManagementMock.Setup(c => c.GetCapabilities(Location))
-                .Returns(propertiesResponse)
-                .Verifiable();
+        // Ignore capablitity related test cases
```

Comment:
> @aim-for-better any reason this test is being removed? If we need to remove it, then we should delete the entire thing rather than commenting it out #Resolved

Remove comments and do some necessary changes to make sure that test case can pass